test/browser/fragments.test.js

```diff
@@ -1295,7 +1295,7 @@ describe('Fragment', () => {
 		expectDomLogToBe([
 			'<ol>012345.insertBefore(<li>4, <li>0)',
 			'<ol>401235.insertBefore(<li>5, <li>0)',
-			// TODO: see issue #193 - Hmmm why does this extra append happen?
+			// TODO: Hmmm why does this extra append happen?
```

Comment:
Unrelated to this PR: Maybe we're wrongly recreating the child here in `diffChildren`. Something to check for another PR :+1: